### PR TITLE
Show outlined progress icons

### PR DESCRIPTION
**Pull request by huayunh**
#### Changes proposed in this Pull Request:
- Added a little switch in the progress icon card to view the outlined icons

#### Screenshots:
image

### CODE REVIEW

**Comment by jeffvg** [comment]
looks good. Wondering how the outlined donuts will actually render on devices. Can we control the outlined weight?

**Comment by huayunh** [comment]
> Can we control the outlined weight?

No. Material design wants all the line strokes in icons to be 2px wide, so perhaps this is where we as pxblue are getting opinionated.

**Comment by joebochill** [comment]
Can we use a checkbox instead of a toggle, like the card above? Or use the toggle with dual labels like we have on the Colors page and put it _in_ the blue toolbar?

**Comment by joebochill** [comment]
> looks good. Wondering how the outlined donuts will actually render on devices. Can we control the outlined weight?

For the donuts specifically, you can adjust the width of the ring, but not the width of the stroke

**Comment by huayunh** [comment]
> put it in the blue toolbar?

That was my first attempt... aborted bc I didn't want to do all that responsive viewport thing 😛

**Comment by huayunh** [comment]
image

File: src/app/components/icons/ProgressIconCard.tsx [hunk 1]
```diff
@@ -34,6 +36,14 @@ export const ProgressIconCard: React.FC = (): JSX.Element => {
                     </Typography>
                 </Toolbar>
             </AppBar>
+            <Toolbar style={{ justifyContent: 'flex-end' }}>
+                <FormControlLabel
+                    control={<Checkbox color="primary" onClick={(): void => setIsOutlined(!isOutlined)} />}
+                    checked={isOutlined}
+                    label={'View Outlined Icon'}
```
Comment (joebochill) [inline src/app/components/icons/ProgressIconCard.tsx:43]:
```suggestion
                    label={'View Outlined'}
```

or View Outlined _Icons_

File: src/app/components/icons/ProgressIconCard.tsx [hunk 2]
```diff
@@ -34,6 +36,14 @@ export const ProgressIconCard: React.FC = (): JSX.Element => {
                     </Typography>
                 </Toolbar>
             </AppBar>
+            <Toolbar style={{ justifyContent: 'flex-end' }}>
```
Comment (joebochill) [inline src/app/components/icons/ProgressIconCard.tsx:39]:
```suggestion
            <Toolbar variant={'dense'} style={{ justifyContent: 'flex-end' }}>
```

File: src/app/components/icons/ProgressIconCard.tsx [hunk 3]
```diff
@@ -34,6 +36,14 @@ export const ProgressIconCard: React.FC = (): JSX.Element => {
...
+                <FormControlLabel
+                    control={<Checkbox color="primary" onClick={(): void => setIsOutlined(!isOutlined)} />}
+                    checked={isOutlined}
+                    label={'View Outlined Icon'}
+                    labelPlacement={'start'}
+                />
+            </Toolbar>
             <div style={{ textAlign: 'center', padding: '24px' }}>
```
Comment (joebochill) [inline src/app/components/icons/ProgressIconCard.tsx:47]:
```suggestion
            <div style={{ textAlign: 'center', padding: '0px 24px 24px' }}>
```

File: src/app/components/icons/IconBrowser.tsx [hunk 4]
```diff
@@ -258,13 +256,13 @@ export const IconBrowser: React.FC = (): JSX.Element => {
                         </div>
                     </Toolbar>
                 </AppBar>
-                <div className={classes.hideIconsLabel}>
+                <Toolbar className={classes.hideIconsLabel}>
```
Comment (joebochill) [inline src/app/components/icons/IconBrowser.tsx:259]:
```suggestion
                <Toolbar variant={'dense'} className={classes.hideIconsLabel}>
```

File: src/app/components/icons/IconBrowser.tsx [hunk 5]
```diff
@@ -258,13 +256,13 @@ export const IconBrowser: React.FC = (): JSX.Element => {
...
                     <FormControlLabel
                         control={<Checkbox color="primary" onClick={(): void => setFilterMaterial(!filterMaterial)} />}
                         label="Hide Material Icons"
                         labelPlacement="start"
                     />
-                </div>
+                </Toolbar>
                 <div style={{ padding: '24px' }}>
```
Comment (joebochill) [inline src/app/components/icons/IconBrowser.tsx:266]:
```suggestion
                <div style={{ padding: '0 24px 24px' }}>
```

**Pull request merged**